pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '9.0'
+lockfileVersion: '6.0'
```

Comment:
if you do not have pnpm v9 currently in use locally, I can generate the file for you